cpp/modmesh/io/gmsh.hpp

```diff
@@ -0,0 +1,397 @@
+#pragma once
+#include <string>
+#include <fstream>
+#include <vector>
+#include <unordered_map>
+#include <functional>
+
+#include <modmesh/base.hpp>
+#include <modmesh/mesh/mesh.hpp>
+#include <modmesh/buffer/buffer.hpp>
+
+namespace modmesh
+{
+namespace IO
+{
+namespace Gmsh
+{
+struct ElementDef
```

Comment:
What about naming it like `GmshElementDef`?

Reply:
I renamed `ElementDef` to `GmshElementDef` in https://github.com/solvcon/modmesh/pull/233/commits/7432534167ee1da998cfb1a7024ed5e8f52f869f